srcpkgs/rhythmbox/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'rhythmbox'
 pkgname=rhythmbox
 version=3.4.4
-revision=2
+revision=3
 build_style=gnu-configure
 build_helper=gir
 configure_args="--disable-static --with-gudev --without-hal
- $(vopt_if gir --enable-vala)
+ $(vopt_if gir --enable-vala) $(vopt_with brasero)
```

Comment:
isn't that autodetected based on `brasero-devel`'s presence?

Reply:
Yeah, `vopt_with` isn't needed, removed it.